x-pack/plugins/infra/public/plugin.ts

```diff
@@ -34,15 +34,16 @@ export class Plugin implements InfraClientPluginClass {
       registerFeatures(pluginsSetup.home);
     }
 
-    pluginsSetup.observability.observabilityRuleTypeRegistry.register(
+    pluginsSetup.triggersActionsUi.ruleTypeRegistry.register(createInventoryMetricAlertType());
```

Comment:
This looks like it undoes a recent change in `master`. Could it be the consequence of an incorrect conflict resolution?

Reply:
Thank you for catching this! I will fix it.